sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -132,15 +132,19 @@ class PlanParserSuite extends AnalysisTest {
       table("a").select(star()).union(table("a").where('s < 10).select(star())))
     intercept(
       "from a select * select * from x where a.s < 10",
-      "Multi-select queries cannot have a FROM clause in their individual SELECT statements")
+      "This select statement can not have FROM cause as its already specified upfront")
```

Comment:
you know what? cannot is correct too and more usual :-).

Reply:
this is copied from the old error message: https://github.com/apache/spark/pull/24348/files#diff-9847f5cef7cf7fbc5830fbc6b779ee10L133

We can update this

Reply:
it's fine. I didn't mean we should fix now. just wanted to say it :-).